testing/simulated/transformers.go

```diff
@@ -41,7 +41,11 @@ import (
 
 // transformSimulatedBlockToGethBlock converts a simulated execution block into a Geth-style block.
 // It uses the provided transactions and parent beacon root to construct a new execution block header.
-func transformSimulatedBlockToGethBlock(simBlock *execution.SimulatedBlock, txs []*gethtypes.Transaction, parentBeaconRoot libcommon.Root) *gethtypes.Block {
+func transformSimulatedBlockToGethBlock(
```

Comment:
this changes do not seem related to the config spec. Why are they here?

Reply:
This got introduced as part of this [commit](https://github.com/berachain/beacon-kit/pull/2580/commits/d9a2eca287f356d49759b9fa9d153e31ecc57902). @calbera shall we remove them ?

Reply:
Ahh that’s my bad, my linter always be doing the most. Good to remove!

Reply:
Removed.